shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/token/pojo/EncryptLiteralAssignmentToken.java

```diff
@@ -56,8 +57,13 @@ public String toString() {
         
         private final Object value;
         
+        private final boolean exceptional;
+        
         @Override
         public String toString() {
+            if (exceptional) {
+                return String.format("%s = VALUES(%s)", columnName, value);
```

Comment:
It seems that hard-writing like this can't solve the internal rewriting problem of other functions.